pkg/az/az.go

```diff
@@ -1,13 +1,18 @@
 package az
 
 import (
+	"context"
 	"fmt"
 	"math/rand"
 	"time"
 
+	ec22 "github.com/aws/aws-sdk-go-v2/service/ec2"
```

Comment:
:eyes:

Reply:
There are some other files that have this too, I'm fixing it. (I have had to rebase and resolve conflicts way too many times).